packages/cra-template/template/src/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { App } from './App';
+import { mergeStyles } from '@fluentui/react';
+import reportWebVitals from './reportWebVitals';
+
+// Inject some global styles
+mergeStyles({
+  ':global(body,html,#root)': {
+    margin: 0,
+    padding: 0,
+    height: '100vh',
+  },
+});
+
+ReactDOM.render(<App />, document.getElementById('root'));
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```

Comment:
might this be a bit advanced for a 'starter template' ?

Reply:
just a heads up - this is the default CRA template stuff, not implemented on our own.

Reply:
FYI link to the current template's repo: [microsoft/create-react-app-uifabric](https://github.com/microsoft/create-react-app-uifabric/).

Reply:
Ah well if it's already existing, feel free to resolve. Up to you if you want to open up the can of legacy worms

Reply:
I'd lean towards removing this. I think it's gone in the latest default CRA template too?

Reply:
> I think it's gone in the latest default CRA template too?

it's not. What was removed is service worker, this is new addition since CRA v4. ( we "discussed it" within PR for that CRA existing repo. )
https://github.com/facebook/create-react-app/tree/master/packages/cra-template/template/src

Reply:
Since this was taken verbatim from the existing template, would you agree to leave it as-is in this PR, and refactor in a subsequent one?

Reply:
I'm fine with leaving it for now.